Celery Beat Health Check: Check for Overdue Tasks Accounting for Scheduler Interval (#297)
codingjoe/django-health-check: README.rst

- AWS S3 storage
- Celery task queue
- Celery ping
- Celery Beat Health Check (via `django_celery_beat`)
- RabbitMQ
- Migrations

> View use instructions for contrib health checks in `docs/contrib.rst` and `docs/settings.rst`

Reviewer: ```suggestion
View usage instructions of contrib health checks in `docs/contrib.rst` and `docs/settings.rst`
```
